src/wic.py

```diff
@@ -0,0 +1,71 @@
+    # find rows that start with Total (this includes Total Women, Total Infant and Total Children rows)
+    Total_re = re.compile("Total")
+    # It's not clear specifically what "LA Total" means, but these rows contains the subtotal values for the specific County
+    County_Total_re = re.compile("LA Total")
+    # find rows that start with three digits (these rows contain County ID and name, example: 031 COOK)
+    County_re = re.compile(r"\d\d\d")
+
+    column_names = ["County_ID", "County", "WIC1", "WIC2", "Amer. Indian or Alaskan Native",
+                    "Asian", "Black or African American", "Native Hawaii or Other Pacific Isl.",
+                    "White", "Multi-Racial", "Total Participants", "Hispanic or Latino"]
+
+    # create empty data frame with column names
+    data = pd.DataFrame(columns=column_names)
+
+    with pdfplumber.open(source_pdf_filepath) as pdf:
+    
+        # range() excludes the last value, so add 1 to include the last page
+        for page_num in range(first_page_zero_indexed, last_page_zero_indexed + 1):
+            page = pdf.pages[page_num]
+
+            # extract_text() adds spaces where the horizontal distance between bits of text is greater than x_tolerance
+            # and adds newline characters where the vertical distance between bits of text is greater than y_tolerance.
+            text = page.extract_text(x_tolerance=2, y_tolerance=0)
+
+            # iterate thru each line on a page
+            for line in text.split("\n"):
+                if County_re.match(line):
+                    # We have to find the County information first because we insert it in every row
+                    # maxsplit=1 because some counties have spaces in their name, example: Jo Daviess
+                    County = (line.split(sep=" ", maxsplit=1)) 
+                elif Total_re.match(line):
+                    new_line = (line.split(sep=" ")) # This looks something like ["Total", "Women", 1, 2, 3, 4]
+                    new_line = County + new_line
+                    data = data.append(pd.Series(new_line, index=data.columns), ignore_index=True)
+                elif County_Total_re.match(line):
+                    new_line = (line.split(sep=" ")) # This looks something like ["LA", "Total", 1, 2, 3, 4]
+                    new_line = County + new_line
+                    data = data.append(pd.Series(new_line, index=data.columns), ignore_index=True)
+
+    # Currently the data looks like this:
+    # WIC1      WIC2       etc
+    # Total     Women
+    # Total     Children
+    # LA        Total
+
+    # We want to combine WIC1 and WIC2 columns into new column called WIC which will contain values such as "Total Women"
+    # "Total Children" "Total Infants" and "LA Total"
+    data.insert(2, "WIC", (data["WIC1"] + " " + data["WIC2"]))
+
+    # delete WIC1 and WIC 2 columns
+    data.drop(['WIC1', 'WIC2'], axis=1, inplace=True)
+
+    data.to_csv(destination_csv_filepath, index=False)
```

Comment:
Right now we take the dataframe and just export it to CSV. How does this data need to be integrated with the Census data?

Reply:
The index needs to be the County FIPs to merge in